packages/webapp/pages/tags/[tag].tsx

```diff
@@ -164,15 +170,16 @@ const TagPage = ({ tag, initialData }: TagPageProps): ReactElement => {
       return 'unfollowed';
     }
     if (
-      feedSettings.blockedTags?.findIndex((blockedTag) => tag === blockedTag) >
-      -1
+      feedSettings.blockedTags &&
+      feedSettings.blockedTags.findIndex((blockedTag) => tag === blockedTag) >
+        -1
     ) {
       return 'blocked';
     }
     if (
-      feedSettings.includeTags?.findIndex(
-        (includedTag) => tag === includedTag,
-      ) > -1
+      feedSettings.includeTags &&
+      feedSettings.includeTags.findIndex((includedTag) => tag === includedTag) >
+        -1
```

Comment:
Not related to PR but seems like this could be abstracted in a function for this and the above if